### PR TITLE
[8.8] Revert "docs: temp comment out include (#2626)"

### PR DESCRIPTION
**Pull request by bmorelli25**
This reverts commit 413a91f502c4d6fda5c5f841d86e72f3fb039de6.

### CODE REVIEW

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

File: docs/en/install-upgrade/breaking.asciidoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ Upgrade Assistant provides information about which {dfeeds} need to be updated.
 This list summarizes the most important breaking changes in APM.
 For the complete list, go to {apm-guide-ref}/apm-breaking.html[APM breaking changes].
 
-// include::{apm-repo-dir}/apm-breaking.asciidoc[tag=notable-v8-breaking-changes]
+include::{apm-repo-dir}/apm-breaking.asciidoc[tag=notable-v8-breaking-changes]
```
Comment (colleenmcginnis) [inline docs/en/install-upgrade/breaking.asciidoc:36]: I think this needs to be `{apm-repo-dir}/changelogs/all-breaking-changes.asciidoc`, but there's no `notable-v8-breaking-changes` tag in that file. 🤔

Comment (bmorelli25) [reply docs/en/install-upgrade/breaking.asciidoc:36]:
```suggestion
No notable breaking changes.
```

File: docs/en/install-upgrade/breaking.asciidoc [hunk 2]
```diff
@@ -33,7 +33,7 @@ Upgrade Assistant provides information about which {dfeeds} need to be updated.
 This list summarizes the most important breaking changes in APM.
 For the complete list, go to {apm-guide-ref}/apm-breaking.html[APM breaking changes].
 
-// include::{apm-repo-dir}/apm-breaking.asciidoc[tag=notable-v8-breaking-changes]
+No notable breaking changes.
```
Comment (bmorelli25) [inline docs/en/install-upgrade/breaking.asciidoc:36]: Updating to the verbage we use elsewhere.
```suggestion
There are no breaking changes in APM.
```

**Pull request merged**